packages/client-config/src/client-config-contributor/client_config_contributor_v1.test.ts

```diff
@@ -162,9 +164,11 @@ void describe('auth client config contributor v1', () => {
 
 void describe('data client config contributor v1', () => {
   void it('returns an empty object if output has no graphql output', async () => {
-    const modelSchemaAdapter = new ModelIntrospectionSchemaAdapter(
-      fromNodeProviderChain()
-    );
+    const modelSchemaAdapter = new ModelIntrospectionSchemaAdapter({
+      getS3Client: () => new S3Client(),
+      getAmplifyClient: () => new AmplifyClient(),
+      getCloudFormationClient: () => new CloudFormationClient(),
```

Comment:
might be helpful to create a `stubClientProvider` object so that this doesn't have to be repeated so much in the tests (applies to other test files as well)